libr/debug/p/native/linux/linux_coredump.c

```diff
@@ -1321,6 +1319,7 @@ static void init_note_info_structure(int pid, size_t auxv_size) {
 	/* NT_X86_XSTATE_T*/
 	type = NT_X86_XSTATE_T;
 	note_info[type].size = get_xsave_size (pid);
+	eprintf ("NT_X86_XSTATE_T: %d\n", note_info[type].size);
```

Comment:
i would remove this eprintf since i think you forgot while you were debugging, for the rest lgtm